scenario/migrate.yaml

```diff
@@ -42,6 +42,7 @@ process:
           - init_iteration_instance_copy_var: True
           - init_iteration_instance_ref: True
       - check_needed_compute_resources: True
+      - check_instances: ['rename_info_iter1']
```

Comment:
It looks like in this case we will not skip all transport instances tasks and will go to the next task (`get_next_instance`) if there are no instances on the source. I guess we need to do the same as in the os2os, namely use the `rename_info_iter`.